package-lock.json

```diff
@@ -0,0 +1,3932 @@
+{
```

Comment:
Please remove `package-lock.json`, we are intentionally NOT using that feature of `npm`.